wsdl/ver10/device/wsdl/devicemgmt.wsdl

```diff
@@ -394,6 +399,41 @@ IN NO EVENT WILL THE CORPORATION OR ITS MEMBERS OR THEIR AFFILIATES BE LIABLE FO
 				</xs:restriction>
 			</xs:simpleType>
 			<!--===============================-->
+			<xs:simpleType name="HardwareTypes">
+				<xs:restriction base="xs:string">
+					<xs:enumeration value="Camera">
+						<xs:annotation>
+							<xs:documentation>Single sensor camera. Ex. bullet, PTZ or fisheye camera.</xs:documentation>
+						</xs:annotation>
+					</xs:enumeration>
+					<xs:enumeration value="Multi-sensor">
+						<xs:annotation>
+							<xs:documentation>Multiple sensors device.</xs:documentation>
+						</xs:annotation>
+					</xs:enumeration>
+					<xs:enumeration value="Encoder">
+						<xs:annotation>
+							<xs:documentation>Analog encoder. Can have 1 or more encoders.</xs:documentation>
+						</xs:annotation>
+					</xs:enumeration>
+					<xs:enumeration value="Intercom">
+						<xs:annotation>
+							<xs:documentation>Intercom device, which has both a microphone and a speaker.</xs:documentation>
+						</xs:annotation>
+					</xs:enumeration>
+					<xs:enumeration value="AccessControl">
+						<xs:annotation>
+							<xs:documentation>Access control device.</xs:documentation>
+						</xs:annotation>
+					</xs:enumeration>
+					<xs:enumeration value="NetworkSpeaker">
+						<xs:annotation>
+							<xs:documentation>A network speaker which is not an intercom.</xs:documentation>
```

Comment:
Suggestion to extend this list with Recorder, NetworkMic, Display (network display supporting mainly display service) and IO-Device (Supporting mainly IO service).

Reply:
Could you provide a description of what a Recorder is?